src/AddressRegistry.Api.Oslo/AddressMatch/AddressMatchContext.cs

```diff
@@ -0,0 +1,66 @@
+namespace AddressRegistry.Api.Oslo.AddressMatch
```

Comment:
Don't think addressmatch is needed atm

Reply:
They wanted AddressMatch to be V2 as well but making it Oslo compliant is still in refinement stage